apiserver/facades/client/application/application.go

```diff
@@ -2166,6 +2171,62 @@ func (api *APIBase) consumeOne(arg params.ConsumeApplicationArg) error {
 	return err
 }
 
+// Consume adds remote applications to the model without creating any
+// relations.
+func (api *APIv19) Consume(args params.ConsumeApplicationArgsV4) (params.ErrorResults, error) {
```

Comment:
We don't need to duplicate all this code. The whole point of embedding APIv20 in APIv19 is to allow this just to call `api.APIv20.Consume(...)` (after converting the args etc)

Reply:
Of course! this was the correct way to do it from the beginning. Thanks! fixed...